src/custom-elements/store.js

```diff
@@ -1,23 +1,42 @@
-import { reactive } from 'vue';
+import { computed, reactive } from 'vue';
 
+const settings = reactive({});
 const store = reactive({ packages: {} });
 window.__overlay_global_store = store;
 
-export const updatePackageInfo = ({ type, name }, part, info) => {
-  if (!store.packages[type]) {
-    store.packages[type] = {};
-  }
-  if (!store.packages[type][name]) {
-    store.packages[type][name] = {
+const packageStoreID = ({ type, name }) => `${type}$${name}`;
```

Comment:
I avoided using `/` because it is an allowed character in packages like `@scoped/package`.

I remember I saw a "package identifier convention" in one of the advisories, I will search for that.

Reply:
Can we use `:` instead of `$$` or `/`?

Reply:
yes `:` is also ok

Reply:
done